data/strings/strings.txt

```diff
@@ -1140,47 +1140,49 @@
     zh-Hans = 由爱好者和志愿者创建的开源应用程序。
     zh-Hant = 由愛好者和志願者創建的開源應用程序。
 
-  [connection_settings]
+  [location_settings]
     tags = android
-    en = Connection Settings
-    af = Verbindingsinstellings
-    ar = إعدادات الاتصال
-    be = Налады падключэння
-    bg = Настройки за свързване
-    ca = Paràmetres de connexió
-    cs = Nastavení připojení
-    da = Forbindelsesindstillinger
-    de = Verbindungseinstellungen
-    el = Ρυθμίσεις σύνδεσης
-    es = Ajustes de conexión
-    et = Ühenduse seaded
-    eu = Konexio ezarpenak
-    fa = تنظیمات اتصال
-    fi = Yhteysasetukset
-    fr = Paramètres de connexion
-    he = הגדרות תקשורת
-    hu = Helymeghatározási beállítások
-    id = Pengaturan Koneksi
-    it = Impostazioni di connessione
-    ja = 接続設定
-    ko = 연결 설정
-    lt = Tinklo nustatymai
-    mr = जोडणी सेटिंग
-    nb = Tilkoblingsinnstillinger
-    nl = Verbindingsinstellingen
-    pl = Ustawienia połączenia
-    pt = Definições de ligação
-    pt-BR = Configurações de conexão
-    ro = Opțiuni de conectare
-    ru = Настройки подключения
-    sk = Nastavenie pripojenia
-    sv = Anslutningsinställningar
-    th = การตั้งค่าการเชื่อมต่อ
-    tr = Konum Ayarları
-    uk = Налаштування підключення
-    vi = Thiết lập Kết nối
-    zh-Hans = 连接设置
-    zh-Hant = 連線設定
+    comment = The button that opens system location settings
+    en = Location Settings
+    ar = إعدادات الموقع
+    be = Налады месцазнаходжання
+    bg = Настройки на местоположението
+    ca = Configuració d'ubicació
+    cs = Nastavení polohy
+    da = Indstillinger for placering
+    de = Standort-einstellungen
```

Comment:
de = Standort Einstellungen

Reply:
"Standort Einstellungen" (space and capital E) doesn't look right. That's two unconnected words instead of one term.
I think either
"Standorteinstellungen"
or better (for readability)
"Standort-Einstellungen"

Reply:
you're right. Standorteinstellungen would be correct, I capitalized it for better readability.